core/domain/html_cleaner.py

```diff
@@ -25,19 +25,16 @@
 
 
 def filter_a(name, value):
-    """Filters the target value from the link.
+    """Returns whether the given attribute of an 
+    anchor ('a') tag should be whitelisted.
```

Comment:
Why are we wrapping this before 80 chars? Ditto below (in at least two cases).

Btw, if you're using sublime text, you can add 

```
{
	"rulers":
	[
		80
	]
}
```

to your user preferences (Preferences > Settings - User) to introduce a vertical line at 80 chars in your editor.